src/benchmark/tpch_benchmark.cpp

```diff
@@ -165,6 +169,33 @@ class TpchBenchmark final {
     } else {
       _create_report(std::cout);
     }
+
+    /**
+     * Visualize query plans
+     */
+    if (_enable_visualization) {
```

Comment:
This looks like a weird thing to do in the run method. I think I would have expected the main method to call this once after the command line arguments have been parsed. That would also save us from having that class member

Reply:
I want the TpchBenchmark functionality to be a thing that can be easily "automated" using C++ code, i.e. run with different configuration. Doing the visualization in main() would break this.